Cargo.toml

```diff
@@ -332,9 +332,9 @@ datafusion-table-providers = { git = "https://github.com/datafusion-contrib/data
 
 datafusion-federation = { git = "https://github.com/spiceai/datafusion-federation.git", rev = "afcbd1ce99703f1322d176fc1b99745f647234d0" } # spiceai:spiceai-50
 
-ballista-core = { git = "https://github.com/spiceai/datafusion-ballista.git", rev = "12957e480d1433b97bbb5de328e6bc0387d8551a" }      # spiceai/spiceai-50
-ballista-executor = { git = "https://github.com/spiceai/datafusion-ballista.git", rev = "12957e480d1433b97bbb5de328e6bc0387d8551a" }  # spiceai/spiceai-50
-ballista-scheduler = { git = "https://github.com/spiceai/datafusion-ballista.git", rev = "12957e480d1433b97bbb5de328e6bc0387d8551a" } # spiceai/spiceai-50
+ballista-core = { git = "https://github.com/spiceai/datafusion-ballista", rev = "123697fdef3459f184ac8846951d5932c0a3b941" }           # spiceai/spiceai-50
```

Comment:
TODO: update to use final commit when https://github.com/spiceai/datafusion-ballista/pull/3 is merged